packages/truffle-compile-vyper/index.js

```diff
@@ -83,8 +83,11 @@ function checkVyper(callback) {
 }
 
 // Execute vyper for single source file
-function execVyper(source_path, callback) {
+function execVyper(options, source_path, callback) {
   const formats = ["abi", "bytecode", "bytecode_runtime"];
+  if (options.vyper && options.vyper.source_map) {
```

Comment:
So I think we want to use camelCase for sourceMap (for consistency within truffle-config.js),
and we also probably want to have the config setting be under options.compilers, rather than start a separate key for vyper.

```suggestion
  if (options.compilers.vyper.sourceMap) {
```